tests/momento/requests/vector_index/test_filters.py

```diff
@@ -0,0 +1,131 @@
+import pytest
+from momento.requests.vector_index import Field, FilterExpression
+from momento.requests.vector_index import filters as F
+from momento_wire_types import vectorindex_pb2 as pb
+
+
+@pytest.mark.parametrize(
+    ["field_based_expression", "full_expression", "protobuf"],
+    [
+        (
+            Field("foo") == "bar",
+            F.Equals("foo", "bar"),
+            pb._FilterExpression(equals_expression=pb._EqualsExpression(field="foo", string_value="bar")),
+        ),
```

Comment:
See this test file for many examples of field expression syntax using either a functional style or idiomatic style with operator overloading.